loopdb/sqlite.go

```diff
@@ -342,6 +342,26 @@ func fixTimeStamp(dateTimeStr string) (time.Time, error) {
 		)
 	}
 
+	// If the year is a leap year and the date is 29th of February, we
+	// need to change it to 28th of February. Otherwise, the time.Parse
+	// function will fail, as a non-leap year cannot have 29th of February.
+	day, month, err := extractDayAndMonth(dateTimeStr)
+	if err != nil {
+		return time.Time{}, fmt.Errorf("unable to parse timestamp day "+
+			"and month %v: %v", dateTimeStr, err)
+	}
+
+	if !isLeapYear(thisYear) && isLeapYear(year) &&
```

Comment:
couldn't we just replace this if `thisYear` is not a leap year?
`if !isLeapYear(thisYear) && month == 2 && day == 29 {...`